src/main/resources/db/changelog/changes/workout_data/v1.0-workout-changelog.sql

```diff
@@ -27,7 +27,7 @@ CREATE TABLE workout_data.routine
     modified_by UUID                     NOT NULL,
     user_id     UUID                     NOT NULL,
     name        VARCHAR(255)             NOT NULL,
-    description VARCHAR(255),
+    description VARCHAR(3000),
```

Comment:
this change is causing liquibase to fail validation, in such cases the proper approach should be to create another changelog to modify the field

Reply:
true, i mean we are in early stages of the dev so its not problem to drop the tables, but i will make it proper for convenience :)